frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -82,92 +80,96 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
     );
   }
 
+  const panelContent = selectedNode ? (
+    <SelectedTaskDrawerContent
+      task={selectedNode.data.pipelineTask}
+      onClose={() => setSelectedId(null)}
+    />
+  ) : null;
+
```

Comment:
why make this conditional. it looks like you can set if the panel is open in the viz surface

Reply:
Updated - thank you @Gkrumbach07 for your help this afternoon in resolving this